src/components/Navigator/NavigatorTab.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { styled } from "@stitches/react";
+import { InternationalString } from "@hyperion-framework/types";
+import { LabeledResource } from "hooks/use-hyperion-framework/getContentResourcesByCriteria";
+import { getLabel } from "hooks/use-hyperion-framework";
+
+interface NavigatorTabProps {
+  active: boolean;
+  resource: LabeledResource;
+  handleChange: (arg0: string) => void;
+}
+
+const NavigatorTab: React.FC<NavigatorTabProps> = ({
+  active,
+  resource,
```

Comment:
How heavy to you imagine `resource` to be?   If a WebVTT contains a transcript of thousands of lines say, would that content come along with the value for `resource`?   Just wondering if we could only pass in `id` and `label` off `resource`...